src/parse/call.rs

```diff
@@ -77,14 +105,37 @@ mod test {
     use crate::parse::{parse, parse_direct};
     use crate::parse::ast::{AST, Node};
 
+    #[test]
+    fn op_assign() {
```

Comment:
Remainder should be tested in an end-to-end test ideally.
Separate test to see if the correct Python code is generated for this AST is misleading as it always goes through a type checker first.